client/my-sites/importer/importer-wordpress.jsx

```diff
@@ -39,7 +39,7 @@ export default React.createClass( {
 	render: function() {
 		importerData.description = this.translate(
 			'Import posts, pages, and media ' +
-			'from a WordPress export file.'
+			'from a WordPress export\u00A0file.'
```

Comment:
See `preventWidows` in <a href="https://github.com/Automattic/wp-calypso/blob/master/client/lib/formatting/index.js#L69">formatting utilities</a> in case it's helpful.